### DISCUSSION TITLE
how can i save in ANSI ?

### DISCUSSION THREAD

**Discussion by Bonjir** [0x7c13/Notepads#1186] [General]
i can't find an option to save my files in ascii, but saving in acsii is vital for me to write the bat script files

**Comment by 0x7c13** [comment]
![image](https://user-images.githubusercontent.com/1166162/215145712-ef81e9bf-86d9-42e6-8071-5c29d691f025.png)
Whatever displayed here at the top is your "ANSI" encoding.
